py27googledriveconnect/googledriveutils.py

```diff
@@ -0,0 +1,164 @@
+    # Navigate through the directories
+    wlab_fid = find_folderid('Winkler Lab', 'root')
+    rdata_fid = find_folderid('ReactorData', wlab_fid)
+    try:  # When we find reactordata directory, look for our reactor's folder
+        r_fid = find_folderid(r_folder_name,rdata_fid)
+        return r_fid
+    except: # If we can't find it, say so.
+        raise NoSuchReactor('There is no reactor with that number')
+
+
+def find_reactorfileid(reactorno, filename):
+    """
+    Find a specific file withn a specified reactor's directory
+    :param reactorno: int, the number of reactor in question
+    :param filename: str, the name of file being looked for
+    :return: file, this is the file
+     OR
+    :return: boolean, this is false if there is no file matching specified name.
+                      The boolean is then used in write_to_reactordrive to make a new file
+    """
+    tgt_folder_id = find_reactorfolder(reactorno)  # Get file id of our reactor's folder
+    file_list = get_file_list(tgt_folder_id) # List all files in that folder
+    no_file = True
+    for afile in file_list:
+        if afile['title'] == filename:  # Look for file that matches name
+            no_file = False # Now we know that file exists, so we don't need to make it later
+            fileid = afile
+            return fileid # Return our file of interest
+    if no_file: # If the file we asked for doesn't exist, tell next function with a boolean.
+        return False
+
+
+def write_to_reactordrive(reactorno, filename, text):
+    """
+    Writes some specified info to a specified file for a specified reactor
+    :param reactorno: int, this is the reactor in question
+    :param filename: this is the name of the file we want to write to.
+    :param text: this is the data we want to write
+    """
+    file_to_write = find_reactorfileid(reactorno, filename)  # Find our file we asked for
+    if file_to_write is False:  # Create a new file if file doesn't exist
+        tgt_folder_id = find_reactorfolder(reactorno)  # Find the id of directory we want to save to.
+        file_to_write = drive.CreateFile({'title': filename, 'mimeType':'text/csv',
+            "parents": [{"kind": "drive#fileLink","id": tgt_folder_id}]})  # Make that file
+    file_to_write.SetContentString(text) # Put the content we want in the file
+    file_to_write.Upload()  # Upload it
+
+
+def read_from_reactordrive(reactorno, filename, save_file_as):
+    """
+
+    :param reactorno: int, this is the reactor in question
+    :param filename: this is the name of the file we want to read
+    :param save_file_as: str, this is what you want to save the file as.
+    :return:
+    """
+    file_to_read = find_reactorfileid(reactorno, filename) # find file we asked for
+    if file_to_read is False:
+        raise NotMatching('The file specified: '+filename+' does not exist')
+    else:
+        file_to_read.GetContentFile(save_file_as)
```

Comment:
Spacing needed after this line? Not clear whether the comments below belong to the function or not. Just some thoughts :)